packages/core/minos-microservice-networks/minos/networks/discovery/connectors.py

```diff
@@ -79,10 +79,7 @@ def _client_from_config(cls, config: Config) -> DiscoveryClient:
         discovery_config = config.get_discovery()
 
         client_cls = cls._client_cls_from_config(discovery_config)
-        client_host = discovery_config.get("host")
-        client_port = discovery_config.get("port")
-
-        return client_cls(host=client_host, port=client_port)
+        return client_cls.from_config(config)
```

Comment:
Could you add a test to check this code?

Reply:
Done!